### ISSUE TITLE
Verify we do not include any tooling dot `.` configuration files in source release

### ISSUE THREAD

**Issue by mrutkows** [apache/incubator-openwhisk-release#161]
Please make sure we do not include the following files which are for config. tooling:
```
.bin
.dockerignore
.git
.gitattributes
.github
.gitignore
.jshintrc
.pydevproject
.rat-excludes
```
or similar files prefixed with a dor `.` character.

**Comment by mrutkows** [comment]
@houshengbo Can we confirm that all TAR files we produce for source release have their "tests" folders/directory trees deleted from final packaging?

**Comment by mrutkows** [comment]
@houshengbo if you have confirmed this by inspection of a TARs you generated for the repos. then feel free to close this issue.  I would appreciate a link to the PR (and file) where you fixed (and configured) this.

**Comment by houshengbo** [comment]
@mrutkows .dockerignore has to stay, because we need to build the docker images locally based on it to determine what are copied into the images.

**Comment by houshengbo** [comment]
This issue is closed, since #182 is merged.
